lib/rules/no-descending-specificity/index.js

```diff
@@ -137,17 +142,27 @@ function lastCompoundSelectorWithoutPseudoClasses(selectorNode) {
 	const nodesAfterLastCombinator = nodesByCombinator[nodesByCombinator.length - 1];
 
 	assert(nodesAfterLastCombinator);
-	const nodesWithoutPseudoClasses = nodesAfterLastCombinator
-		.filter((node) => {
-			return (
-				node.type !== 'pseudo' ||
-				node.value.startsWith('::') ||
-				pseudoElements.has(node.value.replace(/:/g, ''))
-			);
-		})
-		.join('');
 
-	return nodesWithoutPseudoClasses.toString();
+	const nodesWithoutPseudoClasses = nodesAfterLastCombinator.filter((node) => {
+		return (
+			isCssModulesGlobal(node) ||
+			node.type !== 'pseudo' ||
+			node.value.startsWith('::') ||
+			pseudoElements.has(node.value.replace(/:/g, ''))
+		);
+	});
+
+	if (nodesWithoutPseudoClasses.length === 0) return undefined;
```

Comment:
[note] If `nodesWithoutPseudoClasses` is empty, the old version of this function returns an empty string (`''`). This makes it more difficult to handle irregular values. So the current code returns `undefined` instead.